star-wars-app/star-wars-app.js

```diff
@@ -2,12 +2,89 @@ $(document).ready(function() {
 
     // Tatooine is selected on the dropdown by default
     var selectedPlanet = "tatooine";
+    var planetPopulation;
 
     // When the selection on the dropdown changes...
     $("select.planet-selection").change(function(){
         // Store the value of the selected planet into a variable
         selectedPlanet = $(this).children("option:selected").val();
     });
+
+    function generatePopulationGraphic(pop) {
+
+        $(".population-graphic").empty();
+
+        if (selectedPlanet === "alderaan") {
+            $("#1000000000-population").append("<img src=\"images/populationdead.png\" style=\"height: 30px; width: 30px;\">");
+            $("#1000000000-population").append("<img src=\"images/populationdead.png\" style=\"height: 30px; width: 30x;\">");
+        } else if (pop === "unknown") {
+            $("#unknown-population").append("<img src=\"images/populationunknown.png\" style=\"height: 50px; width: 50px;\">");
+        } else {
+        for (i = 100000000000; i >= 100; i /= 10) {
+            if (pop / i >= 1) {
+                var numberOfIcons = pop / i;
+                for (j = 1; j <= numberOfIcons; j++) {
+                    $("#" + i + "-population").append("<img src=\"images/population" + i + ".png\" style=\"height: 25px; width: 25px;\">");
+                }
+                pop = pop % i;
+            }
+        }
+    }
+
+        // if (pop === "unknown") {
```

Comment:
Perhaps for the sake of what is in the master branch, we remove the commented code.  For the presentation, we can show a copy of this file from from your local repo in order to illustrate the changes/evolution of the code.